doc/nrf/doc_build.rst

```diff
@@ -81,145 +90,64 @@ Complete the following steps to build the documentation output:
 
    .. code-block:: console
 
-      cmake -GNinja -B_build .
+      cmake -GNinja -S. -B_build
 
 #. Enter the generated build folder:
 
    .. code-block:: console
 
       cd _build
 
-#. Run ninja to build the documentation:
+#. Run ninja to build the complete documentation:
 
    .. code-block:: console
 
       ninja
 
-   This command will build all documentation sets.
-   Note that this process can take quite some time.
-
-   Alternatively, if you want to build each documentation set separately, complete the following steps:
-
-   a. Run ninja to build the nrfx documentation:
-
-      .. code-block:: console
-
-         ninja nrfx
-
-
-   #. Run ninja to build the Kconfig documentation:
-
-      .. code-block:: console
-
-         ninja kconfig-html
-
-   #. Run ninja to build the Zephyr documentation:
-
-      .. code-block:: console
-
-         ninja zephyr
-
-      This step can take up to 15 minutes.
-
-   #. Run ninja to build the mcuboot documentation:
-
-      .. code-block:: console
-
-         ninja mcuboot
-
-   #. Run ninja to build the nrfxlib inventory file (used by nrf):
-
-      .. code-block:: console
-
-         ninja nrfxlib-inventory
-
-   #. Run ninja to build the |NCS| documentation:
-
-      .. code-block:: console
+The documentation output is written to the :file:`doc/_build/html` folder.
+Double-click the :file:`index.html` file to display the documentation in your browser.
 
-         ninja nrf
+Alternatively, you may also want to work only with a single docset, e.g. ``nrf``.
+The build system provides individual targets for such purpose.
+If you have not built all docsets before, it is recommended to run:
 
-   #. Run ninja to build the nrfxlib documentation:
+.. code-block:: console
 
-      .. code-block:: console
+   ninja ${DOCSET}-html-all
```

Comment:
We usually use the following format:

```
ninja <docset name>-html-all
```

otherwise it looks like it's a real env var